applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml

```diff
@@ -8,10 +8,11 @@ prompt-proto-service:
     repository: ghcr.io/lsst-dm/prompt-proto-service
     pullPolicy: Always
     # Overrides the image tag whose default is the chart appVersion.
-    tag: AuxTel20230524-w_2023_20
+    tag: DM-41084-temporary-latest
 
   instrument:
-    calibRepo: s3://rubin-summit-users/
+    pipelines: (survey="SURVEY")=[${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml]
+    calibRepo: s3://rubin-pp-users/central_repo/
```

Comment:
I wonder if this is why we could read Gaia DR3 on `-dev`, given that `-prod` uses `/app/butler`...

Reply:
Would it be possible to set the `calibRepo` in `values.yaml` to `""`, since this is something we always want to be different between `-dev` and `-prod`?

Reply:
I don't think using the pod's own `/app/butler/butler.yaml` or directly using `/repo/embargo` would make a difference in retrieving a Gaia DR3 file existing on the disk that is not accessible from the pod.  The `butelr.yaml` config are the same. The registry records had paths like `file:///sdf/data/rubin/shared/the_monster/GAIA_DR3`  and retrieving unaccessible files from inside the pod would just never work. 

Anyhow, [DM-41171](https://jira.lsstcorp.org/browse/DM-41171) will disable using the workaround in `prod`.